ros2pkg/ros2pkg/build_type/README.md

```diff
@@ -0,0 +1,231 @@
+# ROS2 Package Build-Types
```

Comment:
It appears you were in the pkg root directory, i.e., ament/, when you ran the `mkdir` command. The command should be run from ament_x/ament_x directory.   Here's the snippet from the doc: 
```
cd <package_root_dir>/ament_x/
mkdir -p resource/ament_x
touch resource/ament_x/main.x.em
```
I can see how this may be confusing. I'm making updates to the tutorial now and will improve that section of the tutorial to be much more clear to the reader.

Reply:
Aha, that makes sense. I'll test again. Thanks for clarifying.